lib/Helper/XiboUploadHandler.php

```diff
@@ -327,6 +327,11 @@ protected function handle_form_data($file, $index)
             $file->enableStat = $media->enableStat;
             $file->mediaType = $module->getModuleType();
 
+            // we need original fileName returned for video thumbnails logic.
+            if ($media->mediaType == 'video') {
```

Comment:
we could always return this - so that the thumbnail function can be used generically.